go-mode.el

```diff
@@ -211,6 +211,14 @@ mis-identifying them as gb projects."
   :type '(repeat function)
   :group 'go)
 
+(defcustom go-confirm-playground-uploads t
+  "Ask before uploading code to the public go Playground.
```

Comment:
It's `Go`, not `go`. Same typo on line 1292

Reply:
Fixed.